projects/dashboards/src/persistence/store/dashboard-remote-store.ts

```diff
@@ -0,0 +1,64 @@
+/* tslint:disable:no-console */
+import { Injectable } from '@angular/core';
+import { EMPTY, Observable, throwError } from 'rxjs';
+import { catchError, map } from 'rxjs/operators';
+// TODO: fix import
```

Comment:
The purpose of the PR right now is not to get it merged because there is a lot more work remaining. Just wanted you folks to review and have your thoughts on the approach. Should have mentioned this in the description when I created the PR. Have added it now